sr_robot_commander/src/sr_robot_commander/sr_arm_commander.py

```diff
@@ -23,12 +23,15 @@ class SrArmCommander(SrRobotCommander):
     Commander class for arm
     """
 
-    def __init__(self, name="right_arm"):
+    def __init__(self, name="right_arm", set_ground=True):
         """
         Initialize object
         @param name - name of the MoveIt group
+        @param sets the ground plane in moveit for planning
         """
         super(SrArmCommander, self).__init__(name)
+        if set_ground:
+            self._move_group_commander.ground()
```

Comment:
Would it make sense to put this in the base class?

Or is it effectively setting this in a common node?